platform-sdk/docs/base/logging/logging.md

```diff
@@ -0,0 +1,210 @@
+## Hedera Logging API
+
+> [!WARNING]  
+> The API is currently in develpment and is not optimized for production use.
+ It is subject to change without notice.
+
+
+### Introduction
+The Hedera Logging API is a custom logging facade which is crafted to meet our unique requirements, ensuring compatibility with various software integrations.
+It acts as a bridge, redirecting events from eg SLF4J to our facade, providing flexibility for all dependencies used in the project.
+
+
+### Architecture and Configuration
+The architecture does not dictate the format of logging output, allowing for a wide range of output options.
+This flexibility permits various configurations, from console outputs to integrations with systems like Graylog or Kibana.
+
+
+### Features of the Logging Facade
```

Comment:
I would not add the headline but have the content under the architecture headline